R/ProjectConfiguration.R

```diff
@@ -0,0 +1,78 @@
+#' @title ProjectConfiguration
+#' @docType class
+#' @description An object storing configuration used project-wide
+#' @format NULL
+ProjectConfiguration <- R6::R6Class(
+  "ProjectConfiguration",
+  inherit = ospsuite.utils::Printable,
+  cloneable = TRUE,
+  active = list(),
+  private = list(),
+  public = list(
+    #' @description
+    #' Initialize a new instance of the class
+    #' @return A new `ProjectConfiguration` object.
+    initialize = function() {
+    },
```

Comment:
You don't need to provide an empty constructor. R6 class will provide a constructor in such cases.

``` r
library(R6)
myClass <- R6Class("myClass")
myClass$new()
#> <myClass>
#>   Public:
#>     clone: function (deep = FALSE)
```

<sup>Created on 2022-04-28 by the [reprex package](https://reprex.tidyverse.org) (v2.0.1)</sup>